x-pack/test/alerting_api_integration/security_and_spaces/group1/tests/alerting/backfill/task_runner.ts

```diff
@@ -167,7 +166,7 @@ export default function createBackfillTaskRunnerTests({ getService }: FtrProvide
       objectRemover.add(spaceId, ruleId, 'rule', 'alerting');
 
       const start = moment(originalDocTimestamps[1]).utc().startOf('day').toISOString();
-      const end = moment().utc().startOf('day').subtract(9, 'days').toISOString();
+      const end = moment(originalDocTimestamps[11]).utc().startOf('day').toISOString();
```

Comment:
fixing the end timestamp so it relates to the doc timestamps and not to the current time.